templates/wg.conf

```diff
@@ -7,11 +7,11 @@
 Address = {{$first :=true}}{{range .serverConfig.Interface.Addresses }}{{if $first}}{{$first = false}}{{else}},{{end}}{{.}}{{end}}
 ListenPort = {{ .serverConfig.Interface.ListenPort }}
 PrivateKey = {{ .serverConfig.KeyPair.PrivateKey }}
-{{if .globalSettings.MTU}}MTU = {{ .globalSettings.MTU }}{{end}}
-PostUp = {{ .serverConfig.Interface.PostUp }}
-PreDown = {{ .serverConfig.Interface.PreDown }}
-PostDown = {{ .serverConfig.Interface.PostDown }}
-Table = {{ .globalSettings.Table }}
+{{if .globalSettings.MTU }}MTU = {{ .globalSettings.MTU }}
+{{end}}{{if .serverConfig.Interface.PostUp }}PostUp = {{ .serverConfig.Interface.PostUp }}
+{{end}}{{if .serverConfig.Interface.PreDown }}PreDown = {{ .serverConfig.Interface.PreDown }}
+{{end}}{{if .serverConfig.Interface.PostDown }}PostDown = {{ .serverConfig.Interface.PostDown }}
+{{end}}{{if .globalSettings.Table }}Table = {{ .globalSettings.Table }}{{end}}
```

Comment:
If am not mistaken, wireguard allows us to have these configs set with an empty value. That's why we didn't care much about removing them completely from the config file, to make it easier to maintain the template. For that reason, I don't think we need if statement here.